src/output/output.hpp

```diff
@@ -25,13 +26,15 @@ using UserDefVariablesFunc = void (*) (DataBlock &, UserDefVariablesContainer &)
 class Output {
   friend class Dump;    // Allow dump to have R/W access to private variables
   friend class Vtk;     // Allow VTK to have access to user-defined variables
+  friend class Xdmf;    // Allow XDMF to have access to user-defined variables
   friend class DumpImage; // Allow dumpimag to have access to dump API
  public:
   Output(Input &, DataBlock &);           // Create Output Object
   int CheckForWrites(DataBlock &);        // Check if outputs are needed at this stage
   void RestartFromDump(DataBlock &, int);  // Restart from a dump file.
   void ForceWriteDump(DataBlock &);            // Force write dumps (needed during an abort)
   void ForceWriteVtk(DataBlock &);            // Force write vtks
+  void ForceWriteXdmf(DataBlock &);          // Force write xdmfs
```

Comment:
Needs to be included on demand.

Reply:
We probably need a master #define flag to enable all of this machinery from Cmake. something like `Idefix_XDMF_OUTPUT`